app/views/account/tasks/_form.html.slim

```diff
@@ -2,5 +2,6 @@
   = f.input_field :section, as: :hidden, value: @task.section
   = f.input :title, placeholder: 'Title', wrapper: :bulma_input, required: true
   = f.input :description, placeholder: 'Description', wrapper: :bulma_input
+  = f.input :files, input_html: { multiple: true }
   = f.submit "Save", class: "button is-link"
-  = link_to 'Back', account_workspace_project_path(@project.workspace_id, @project), class: 'button is-text'
+  = link_to 'Back', :back, class: 'button is-text'
```

Comment:
not a very good idea
this button will fail on invalid form

find why

Reply:
yeah ... it's possible. so put link to actual page (but check with new and edit actions)